packages/app-mobile/components/screens/Note.tsx

```diff
@@ -1141,6 +1141,8 @@ class NoteScreenComponent extends BaseScreenComponent {
 					keyboardAppearance={theme.keyboardAppearance}
 					placeholder={_('Add body')}
 					placeholderTextColor={theme.colorFaded}
+					// need some extra padding for iOS so that the keyboard won't cover last line of the note
```

Comment:
Could you also add a link to the original issue please?

Reply:
Done